artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -3178,138 +3187,7 @@ public Queue createQueue(final SimpleString address,
                             final int consumersBeforeDispatch,
                             final long delayBeforeDispatch,
                             final boolean autoCreateAddress) throws Exception {
-      return createQueue(address, routingType, queueName, filterString, user, durable, temporary, ignoreIfExists, transientQueue, autoCreated, maxConsumers, purgeOnNoConsumers, exclusive, lastValue, lastValueKey, nonDestructive, consumersBeforeDispatch, delayBeforeDispatch, autoCreateAddress, false);
-   }
-
-   private Queue createQueue(final SimpleString address,
```

Comment:
Viewing from a phone, so might be more obvious if using a pc. But where has this method gone with all its logic?

Reply:
There were two different but nearly identical implementations of createQueue(). I consolidated these into a single method. There should be no semantic differences.

Reply:
So afaik, the one being removed is the main one called by bits like config loader etc and invoked more directly by many other code paths, so it would be better to remove the other if any IMO.

Reply:
Thanks, i was going to flag about configurationManaged not being present on the other, but have noticed since you've added it, so it shouldn't be an issue.